.docs/Add-VSTeamVariableGroup.md

```diff
@@ -0,0 +1,138 @@
+<!-- #include "./common/header.md" -->
+
+# Add-VSTeamVariableGroup
+
+## SYNOPSIS
+<!-- #include "./synopsis/Add-VSTeamServiceEndpoint.md" -->
+
+## SYNTAX
+

```

Comment:
You can leave the syntax area blank. When the help is shown it is determined from the attributes of the parameters.  Reviewing the help will let you know if you have all the attributes of the parameters are correct.